AmplifyPlugins/Core/AWSPluginsCore/ServiceConfiguration/AmplifyAWSServiceConfiguration.swift

```diff
@@ -41,3 +42,28 @@ public class AmplifyAWSServiceConfiguration: AWSServiceConfiguration {
         super.init(region: regionType, credentialsProvider: nil)
     }
 }
+
+extension AmplifyAWSServiceConfiguration {
+
+    static var platformMapping: [Platform: String] = [:]
+
+    public static func addPlatform(_ platform: Platform, version: String) {
```

Comment:
This name is ambiguous. It's part of a "service configuration", but we're adding a "platform" but also a version string, with no indication as to its purpose. Can we make its use explicit, as in `addUserAgentPlatform`?

Also I thought we were going to append the additional platform info to the user agent string, so that the UA would include both Amplify iOS and Flutter? Maybe that requirement has changed in other discussions.

Reply:
- Changed to `addUserAgentPlatform`. 

- iOS value will always be present when creating the user agent string. Refer line number 66, where we manually add iOS version before creating the platformInformation.